gk/main.c

```diff
@@ -745,6 +822,22 @@ gk_stage2(void *arg)
 	if (ret < 0)
 		goto cleanup;
 
+	if (ipv6_if_configured(&gk_conf->net->front)) {
+		int i;
+		for (i = 0; i < gk_conf->num_lcores; i++) {
+			unsigned int lcore = gk_conf->lcores[i];
+			ret = register_ipv6_acl(&gk_conf->net->front,
+				lcore, GATEKEEPER_IPV6_ACL_ND);
+			if (ret < 0) {
+				RTE_LOG(ERR, GATEKEEPER,
+					"gk: could not register ND IPv6 ACL for lcore %u\n",
+					lcore);
+				goto cleanup;
+			}
+		}
+		build_ipv6_acls(&gk_conf->net->front);
```

Comment:
We can have blocks registering ACLs at stage 2. All we need is to add the ACL building as the last step of stage 2, what we can accomplish adding a stage 2 call in `main/main.c:main()` just before calling `launch_gatekeeper()`.